README.md

```diff
@@ -1,2 +1,21 @@
 # semester_4_python
 Repository for homeworks in 4th semester in SPbSU programming course
+### Homework №3
+1. **Анализ таблицы**:
```

Comment:
Why does only the `Ticket` column has a `str` type?

Reply:
Also, why do you use `category` for relatives (parents, children and siblings)? This looks like a simple numerical value (because it can be compared, unlike gender or survival).